agent/ecscni/plugin_windows.go

```diff
@@ -52,11 +79,17 @@ func (client *cniClient) setupNS(ctx context.Context, cfg *Config) (*current.Res
 			cniNetworkConfig.Network.Type,
 			cfg.ContainerID)
 		runtimeConfig.IfName = networkConfig.IfName
-		_, err := client.libcni.AddNetwork(ctx, cniNetworkConfig, &runtimeConfig)
+		result, err := client.libcni.AddNetwork(ctx, cniNetworkConfig, &runtimeConfig)
 		if err != nil {
 			return nil, errors.Wrap(err, "add network failed")
 		}
 
+		// We save the result from ecs-bridge setup invocation of the plugin.
+		if strings.EqualFold(ECSBridgeNetworkName, cniNetworkConfig.Network.Name) &&
+			cniNetworkConfig.Network.Type == ECSVPCENIPluginExecutable {
+			ecsBridgeResult = result
```

Comment:
This is confusing. Why only storing/returning this particular `result`s? Also, it ultimately doesn't seem to be used when `setupNS` is called: https://github.com/aws/amazon-ecs-agent/pull/2863/files#diff-e3a9b4000263d327ff4083d797e85e84eca3a2695426bfad0f8c2525ad8f63b7R96

Reply:
discussed offline, to do this change the interface would need to change to return a slice of results, which is not something we are interested in doing now since changes to the linux side would be required as well.